src/components/LoanCards/PromoGridLoanCard.vue

```diff
@@ -46,26 +52,33 @@ export default {
 		},
 	},
 	computed: {
+		imageDensity() {
+			return this.isRetina ? 'retina' : 'std';
+		},
 		targetUrl() {
 			return `/monthlygood?category=${this.experimentData.id}`;
 		},
 		backgroundImage() {
 			try {
 				if (this.experimentVersion === 'shown') {
-					return promoLoanImageRequire(`./mg-promo-${this.experimentData.id}-retina.jpg`);
+					return promoLoanImageRequire(`./mg-promo-${this.experimentData.id}-${this.imageDensity}.jpg`);
 				}
-				// if this.experimentVersion === 'control'
-				return promoLoanImageRequire('./mg-promo-default-retina.jpg');
 			} catch {
-				return promoLoanImageRequire('./mg-promo-default-retina.jpg');
+				// noop, go to default return
 			}
+			// if this.experimentVersion === 'control' or theres some error, return default
+			return promoLoanImageRequire(`./mg-promo-default-${this.imageDensity}.jpg`);
 		}
 	},
 	methods: {
 		trackInteraction(args) {
 			this.$emit('track-interaction', args);
 		}
 	},
+	mounted() {
```

Comment:
Would this work better to establish the retina url if it occurred in created?

Reply:
I was trying to keep it to client side only with mounted, similar how the KvResponsiveImage component does it. I dont think it matters in this particular case, the background image css can be changed on the fly after initial component load without any impact, and our check density utilities just return false if run on the server side.

Reply:
Ok, I'm mostly concerned with double image loading with the standard showing then loading the retina if we detect a higher density...This concern can be separated from this update though as it would affect other locations too.

Reply:
Calling this in created will result in them always being `false` as those check return false if window is not defined. Calling it in mounted is necessary to actually get a retina image at all